lib/ecto_ldap/adapter.ex

```diff
@@ -175,12 +175,14 @@ defmodule Ecto.Ldap.Adapter do
   def execute(_repo, query_metadata, prepared, params, preprocess, options) do
     IO.inspect prepared
     IO.inspect params
-    {:filter, filter} = construct_filter(Keyword.get(prepared, :filter), params)
-    options_filter = :eldap.and(translate_options_to_filter(options))
-    full_filter = :eldap.and([filter, options_filter])
-    full_search_terms = Keyword.put(prepared, :filter, full_filter)
-    IO.inspect(full_search_terms |> replace_dn_search_with_uid_present)
-    search_response   = search(full_search_terms)
+
+    {:filter, filter}  = construct_filter(Keyword.get(prepared, :filter), params)
+    options_filter     = :eldap.and(translate_options_to_filter(options))
+    full_filter        = :eldap.and([filter, options_filter])
+    full_search_terms  = Keyword.put(prepared, :filter, full_filter)
+    final_search_terms = replace_dn_search_with_uid_present(full_search_terms)
```

Comment:
Minor formatting change here.  Organizing this block made thins a little easier for me to read.